Index: linux-6.1.34/arch/arm/mach-alpine/Kconfig
===================================================================
--- linux-6.1.34.orig/arch/arm/mach-alpine/Kconfig
+++ linux-6.1.34/arch/arm/mach-alpine/Kconfig
@@ -22,5 +22,11 @@ config ARCH_ALPINE
 	select ARM_HAS_SG_CHAIN
 	select ARCH_SUPPORTS_BIG_ENDIAN
 	select FORCE_PCI
+	select PCI_HOST_GENERIC
+	select PCI_DOMAINS
+	select PCI_DOMAINS_GENERIC
+	select ARCH_SUPPORTS_MSI
+	select ALPINE_MSI
+	select PCI_MSI_IRQ_DOMAIN
 	help
 	  This enables support for the Annapurna Labs Alpine V1 boards.
